src/main/java/com/aws/greengrass/easysetup/GreengrassSetup.java

```diff
@@ -186,6 +186,9 @@ public class GreengrassSetup {
     private static final String TRUSTED_PLUGIN_ARG = "--trusted-plugin";
     private static final String TRUSTED_PLUGIN_ARG_SHORT = "-tp";
 
+    private static final String CERT_PATH_ARG = "--cert-path";
+    private static final String CERT_PATH_ARG_SHORT = "-cp";
```

Comment:
Don't have add a short version for this, only have the full name for this option.